test/unit/cli_test.py

```diff
@@ -0,0 +1,38 @@
+from argparse import ArgumentTypeError
+
+import pytest
+
+from exasol.toolbox.cli import version
+from exasol.toolbox.release import Version
+
+
+@pytest.mark.parametrize(
+    "version_string,expected_error",
```

Comment:
Maybe add an unexpected number of numbers, e.g. "4.4.4.4"

Reply:
That's an interesting one, for now I have implemented to normalize based on the first three sections separated by a dot. 
I like it.